On stage 1 there is no libdb.so, then we can't build arpd.
Alejandro Mery

--- iproute2-2.6.9/misc/Makefile~	2004-10-19 22:49:02.000000000 +0200
+++ iproute2-2.6.9/misc/Makefile	2004-12-22 21:35:18.037035984 +0100
@@ -1,7 +1,7 @@
 SSOBJ=ss.o ssfilter.o
 LNSTATOBJ=lnstat.o lnstat_util.o
 
-TARGETS=ss nstat ifstat rtacct arpd lnstat
+TARGETS=ss nstat ifstat rtacct lnstat
 
 include ../Config
 
